yoeo/scripts/convertONNXModelToTVM.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import sys
+from shutil import copyfile
+
+import onnx
+
+try:
+    from tvm.driver import tvmc
+    from tvm.driver.tvmc.model import TVMCModel
+    from tvm.relay.transform import ToMixedPrecision
+    from tvm import relay
+except ModuleNotFoundError:
+    print("Please install Apache TVM!")
+    sys.exit(1)
+
+
+def make_parser():
```

Comment:
Maybe add another argument to disable tuning, for example if you just want to create a tvm model for debugging / testing of code